pointcloud/basic_octree_utils_test.go

```diff
@@ -170,6 +170,8 @@ func validateBasicOctree(t *testing.T, bOct *BasicOctree, center r3.Vector, side
 
 	validateMetadata(t, bOct)
 
+	leafMPCMap := make(map[r3.Vector]float64)
```

Comment:
[minor] I don't currently see the need for storing the leafMPCMap if one of the two strategies listed above are used.